models/experimental/functional_unet/tests/test_unet_perf.py

```diff
@@ -10,9 +10,6 @@
 
 from models.perf.perf_utils import prep_perf_report
 from models.perf.device_perf_utils import run_device_perf, check_device_perf, prep_device_perf_report
-from models.utility_functions import (
-    skip_for_grayskull,
-)
 
 from models.experimental.functional_unet.tests.common import UNET_TRACE_REGION_SIZE, UNET_L1_SMALL_REGION_SIZE
```

Comment:
pls lower the l1 usage of unet.
check with Iva if she would like you to do that for SDXL as well.